.travis.yml

```diff
@@ -1,17 +1,24 @@
 lang: ruby
+# Somehow focal (20.04) is available for amd64 but when
+# ran on travis it falls back to xenial (16.04) so just use bionic (18.04)
```

Comment:
Now, after we've migrated to travis-ci.com let's make sure the issue still exists.

Reply:
done